src/www/ui/page/AdminLicenseCandidate.php

```diff
@@ -262,6 +279,32 @@ private function mergeCandidate($candidate, $suggest, $vars)
     return true;
   }
 
+  protected function doDeleteCandidate($rfPk)
+  {
+    $dbManager = $this->getObject('db.manager');
+    $stmt = __METHOD__.".$rfPk";
```

Comment:
`$rfPk` is a argument used in the statement and should not be encoded in the statement name. The name should only depend on the query not the instances of its arguments.